ui/package.json

```diff
@@ -57,9 +57,9 @@
     "primeicons": "^5.0.0",
     "primereact": "^9.6.4",
     "querystring-es3": "^0.2.1",
-    "react": "^17",
+    "react": "18.2.0",
```

Comment:
Can we specify a more flexible range for these, like `^18.2.0` ?

Reply:
I suspect so, but I noticed that Terra-UI uses this exact version.

I reached out to the Terra UI group to see why they went that route.

Reply:
After looking into it, Terra-Ui did this to ensure compatibility between React and React Types.
https://broadinstitute.slack.com/archives/C01EHNUM73R/p1722372032126469

What are your thought here?

Reply:
OK, that works for me.